Dockerfile

```diff
@@ -1,19 +1,44 @@
-FROM ubuntu:21.04
+FROM ubuntu:22.04
 
 ARG DEBIAN_FRONTEND=noninteractive
 ARG QSSTV_CONFIG=qsstv_9.0.conf
 
-RUN sed -i -re 's/ports.ubuntu.com\/ubuntu-ports|security.ubuntu.com\/ubuntu-ports|archive.ubuntu.com\/ubuntu|security.ubuntu.com\/ubuntu/old-releases.ubuntu.com\/ubuntu/g' /etc/apt/sources.list
+#RUN sed -i -re 's/ports.ubuntu.com\/ubuntu-ports|security.ubuntu.com\/ubuntu-ports|archive.ubuntu.com\/ubuntu|security.ubuntu.com\/ubuntu/old-releases.ubuntu.com\/ubuntu/g' /etc/apt/sources.list
 
 RUN apt-get update && apt-get install -y libfftw3-dev libfftw3-3 ffmpeg \
 xvfb qsstv pulseaudio build-essential git libsamplerate0-dev alsa-utils \
-xvfb python3 python3-pip cmake portaudio19-dev python-dev python3-opencv \
-alsa-utils rtl-sdr && rm -rf /var/lib/apt/lists/*
+xvfb python3 python3-pip cmake portaudio19-dev python3-dev python3-opencv \
+alsa-utils rtl-sdr
+
+RUN apt-get -y update && apt-get -y upgrade && apt-get -y install --no-install-recommends \
+    cmake build-essential ca-certificates git libusb-1.0-0-dev \
```

Comment:
really should build in one container and copy over the bins to the second. but its a chonky container anyway.....